src/env_vars.py

```diff
@@ -20,6 +20,9 @@
     "HYDRA_ADMIN_URL": "",
     "IDP_CONFIGMAP_NAME": "",
     "IDP_CONFIGMAP_NAMESPACE": "",
+    "MAIL_HOST": "0.0.0.0",
```

Comment:
These env vars are just placeholder values for now.

Reply:
feel free to ignore: may I suggest to use `"localhost"` as placeholder?
I think this could give the wrong idea that 0.0.0.0 is a proper value